Automattic/jetpack
# Publicize: Sync change to fix the custom excerpt

#### Changes proposed in this Pull Request:
While deploying the fix for #1183 (r232174-wpcom), I noticed that it was a Jetpack sync'd file, so this PR brings the change over to the Jetpack repo.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
This is a code sync, which can't be tested on Jetpack sites. We just need to check that the change is the same as the one committed to wpcom.
